### PR TITLE
Task/images

### PR DESCRIPTION
**Pull request by afoerster**
Better looking diagrams

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ The wizard will ask you for:
 - `smtp address`: The hostname of your smtp server
 - `smtp port`: The port of your smtp server
 
+#### Configuring Sentry for use with Pulse
+Sentry can be used with Pulse to give role based access control to your logs. In order to use
```
Comment (bpmcd) [inline README.md]: Maybe this is just an artifact of the diff, but it looks like the sentence is cutoff.

File: log-appender/src/main/java/io/phdata/pulse/log/HttpAppender.java [hunk 2]
```diff
@@ -107,6 +105,7 @@ private boolean shouldFlush(LoggingEvent event) {
             && lastPostSuccess // the last post was a success
             || currentTime > lastSuccessfulPostTime + backoffTimeSeconds // enough time has passed after the last failure that we want to try to post again
             || event.getLevel().isGreaterOrEqual(Level.ERROR)); // always flush on error messages
+
```
Comment (bpmcd) [inline log-appender/src/main/java/io/phdata/pulse/log/HttpAppender.java]: Extra whitespace.

File: log-appender/src/main/java/io/phdata/pulse/log/HttpAppender.java [hunk 3]
```diff
@@ -73,9 +73,7 @@ protected void append(LoggingEvent event) {
         event.setProperty(HOSTNAME, hostname);
       }
 
-      if (shouldFlush(event)) {
-        flush();
-      }
+      flush();
```
Comment (bpmcd) [inline log-appender/src/main/java/io/phdata/pulse/log/HttpAppender.java]: Flushing the buffer now happens for every logging event?  Also, if you remove `shouldFlush()` from this method, is it used elsewhere in the class? Or should it be removed too?

Comment (afoerster) [reply log-appender/src/main/java/io/phdata/pulse/log/HttpAppender.java]: my master branch was borked, opening a new pull request

**Pull request closed without merging**